src/main/java/com/onaio/steps/modelViewWrapper/HouseholdViewWrapper.java

```diff
@@ -41,7 +42,7 @@ public Household getHousehold(int nameViewId, int numberViewId, int commentsView
         TextView nameView = (TextView) activity.findViewById(nameViewId);
         TextView numberView = (TextView) activity.findViewById(numberViewId);
         String phoneNumber = numberView.getText().toString();
-        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT).format(new Date());
+        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT, Locale.ENGLISH).format(new Date());
```

Comment:
Can I create a new issue for adding support for configurable locales. This was meant to fix an urgent bug where dates were created using the Arabic locale